tests/acceptance/ExportSubscribersCest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Segment;
+
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../_data/MailPoetExportList.csv';
```

Comment:
This test requires the file, but doesn't really do anything with it. There's no valid PHP code in `MailPoetExportList.csv`.
I don't think this `require_once` is necessary here.